sem_3/task_3_6.py

```diff
@@ -0,0 +1,20 @@
+# Пользователь вводит строку текста. Вывести каждое слово с новой строки.
+# ✔ Строки нумеруются начиная с единицы.
+# ✔ Слова выводятся отсортированными согласно кодировки Unicode.
+# ✔ Текст выравнивается по правому краю так, чтобы у самого длинного
+# слова был один пробел между ним и номером строки.
+
+# user_input = input("Введите строку текста: ")
+user_input = """# Пользователь вводит строку текста. Вывести каждое слово с новой строки.
+# ✔ Строки нумеруются начиная с единицы.
+# ✔ Слова выводятся отсортированными согласно кодировки Unicode.
+# ✔ Текст выравнивается по правому краю так, чтобы у самого длинного
+# слова был один пробел между ним и номером строки."""
+
+text = user_input.replace(".", "").replace("#", "").replace(",", "").replace("✔", "").replace("\n", "").replace("  ", " ")\
+    .strip(" ")
+list_from_text = text.split(" ")
```

Comment:
Если нужно разделить строку по пробелам, то лучше ничего не передавать vетоду `split`
```suggestion
list_from_text = text.split()
```